src/ide/vscode/VscodeIDE.ts

```diff
@@ -156,6 +157,29 @@ export default class VscodeIDE implements IDE {
     return this.editorMap.get(editor)!;
   }
 
+  handleCommandError(err: Error) {
```

Comment:
didn't make this part of the `ide` api because we don't really have a need at this point 🤷.  Only accessed from composition root